app/engine/AuditInfo.scala

```diff
@@ -17,34 +17,26 @@
 package engine
 
 import cats.Semigroup
-import engine.RoutingReason.{RoutingReason, _}
+import engine.RoutingReason._
 import model.{Location, Locations}
-import play.api.libs.json.Json.JsValueWrapper
 import play.api.libs.json.{JsObject, Json}
 import play.api.mvc.{AnyContent, Request}
+import uk.gov.hmrc.auth.core.{Enrolment, Enrolments}
 import uk.gov.hmrc.http.HeaderCarrier
 import uk.gov.hmrc.play.audit.AuditExtensions.auditHeaderCarrier
 import uk.gov.hmrc.play.audit.model.ExtendedDataEvent
-import uk.gov.hmrc.play.config.AppName
-import uk.gov.hmrc.play.frontend.auth.AuthContext
-import play.api.Play.current
-
 
 sealed trait TAuditInfo {
   lazy val transactionNames = Map(
     Locations.PersonalTaxAccount -> "sent to personal tax account",
     Locations.BusinessTaxAccount -> "sent to business tax account"
   )
 
-  def toAuditEvent(location: Location)(implicit hc: HeaderCarrier, authContext: AuthContext, request: Request[AnyContent]): ExtendedDataEvent = {
+  def toAuditEvent(location: Location)(implicit hc: HeaderCarrier, authContext: Enrolments, request: Request[AnyContent]): ExtendedDataEvent = {
```

Comment:
should the param authContext be renamed to enrolments?

Reply:
will do